assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.test.js

```diff
@@ -498,4 +477,126 @@ describe( 'AudienceTilesWidget', () => {
 
 		await act( () => waitForTimeouts( 50 ) );
 	} );
+
+	it( 'should show the "no audiences" banner when there is no matching audience', async () => {
+		registry
+			.dispatch( MODULES_ANALYTICS_4 )
+			.setAvailableAudiences( availableAudiences );
+
+		registry.dispatch( CORE_USER ).receiveGetAudienceSettings( {
+			configuredAudiences: [ 'properties/12345/audiences/9' ],
+			isAudienceSegmentationWidgetHidden: false,
+		} );
+
+		const syncAvailableAudiencesEndpoint = new RegExp(
+			'^/google-site-kit/v1/modules/analytics-4/data/sync-audiences'
+		);
+
+		muteFetch( syncAvailableAudiencesEndpoint, [] );
+
+		const { container, getByText, waitForRegistry } = render(
+			<WidgetWithComponentProps />,
+			{
+				registry,
+			}
+		);
+
+		await waitForRegistry();
+
+		act( () => {
+			expect(
+				getByText( /You don’t have any visitor groups selected./ )
+			).toBeInTheDocument();
+
+			expect( container ).toMatchSnapshot();
+		} );
+	} );
+
+	it( 'should show the audience segmentation error widget without "Retry" button, when there is an insufficient permission error while syncing available audiences', async () => {
+		const maybeSyncAvailableAudiencesSpy = jest.spyOn(
+			registry.dispatch( MODULES_ANALYTICS_4 ),
+			'maybeSyncAvailableAudiences'
+		);
+
+		maybeSyncAvailableAudiencesSpy.mockImplementation( () => {
+			registry.dispatch( MODULES_ANALYTICS_4 ).receiveError(
+				{
+					code: 'test-error-code',
+					message: 'Test error message',
+					data: {
+						reason: ERROR_REASON_INSUFFICIENT_PERMISSIONS,
+					},
+				},
+				'syncAvailableAudiences',
+				[]
+			);
+		} );
```

Comment:
Instead of mocking the action, we should be able just to mock the POST request. This would be a simpler and arguably more accurate approach.